survey/4-ai.yml

```diff
@@ -52,11 +53,14 @@ questions:
       - Claude (Anthropic)
       - DALL-E (OpenAI)
       - GitHub Copilot
+      - Cursor
```

Comment:
Gemini is not the new google assistant! Remove Bard.
Also maybe add Meta AI (Llama)

Reply:
done